Button Component/button.css

```diff
@@ -0,0 +1,64 @@
+:root {
+    --color-secondary: #E75A0D;
+    --color-primary: #17183B;
+    --font-white : white;
+  }
+
+*{
+    padding: 0px;
+    margin: 0px;
+    box-sizing: border-box;
+    font: 62.5%;
+}
+
+.button{
+    padding: 0.5rem 0.5rem;
```

Comment:
Improvement: Alternative is { padding: 0.5rem}. it will be same.

Reply:
I will make this changes. Thanks